client-side/src/stories/popup/main.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { usePopupManager } from "react-popup-manager";
+import { Popup } from "./popup";
```

Comment:
notice that files: main.js, popup.jsx, popup.scss, popup.stories.scss
are both in src/popup and src/stories/popup folders.
please decide where they should be located and delete them from the second folder

Reply:
i changed